### DISCUSSION TITLE
should have an option for htmlNode return type

### DISCUSSION THREAD

**Discussion by paras-verma** [dicebear/dicebear#186] [Ideas]
createAvatar return a string instead of htmlNode and has an option for base64 so why not one for an html nodes too?

![image](https://user-images.githubusercontent.com/56761068/147365263-361e5bd6-aa79-4163-bf76-9f54c521fa6f.png)

**Comment by FlorianKoerner** [comment]
The library should work in the browser and in NodeJS. For this reason, dicebear does not offer an option for HTML nodes.

But you can easily convert the string to HTML nodes:

```js
const avatar = createAvatar(style, options);
const node = new DOMParser().parseFromString(avatar, "text/xml").firstChild;
```

However, I do not recommend this approach. It is not fast and secure. Instead, I would always recommend the `dataUri` option. (The `base64` option is deprecated and should not be used anymore. Use `dataUri` instead.)

```js
const node = document.createElement('img');
node.src = createAvatar(style, { dataUri: true });
```
